subt/docker/robotika/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM robotika/subt-base:2020-05-06
 
+RUN sudo apt-get update && sudo apt install -y ros-melodic-teleop-twist-keyboard \
+                        mc
```

Comment:
If you are referring to `mc` I know @pavliik likes it and that is why it is already in our base image. No need to add it here.

Reply:
I've resolved the other discussion so that we don't spam other people.

> @zwn : Do we want such things in the base image, or here? What is the rule of thumb?

The rule of thumb is that if it is used in cloudsim, it really really should be in the base image. If it is something we will keep for a long time, it should go there as well. For this teleop, I don't mind keeping it here for now, since I expect larger update of the installed ros packages anyway. At that point I'd do bulk-move to subt-base. 

> @zwn: Do we want to try pinned versions? Or rather not, because, for example, the specific version may become unavailable?

I prefer using the base image for this purpose.

> @pavliik : –no-install-recommends will help us avoid unnecessary packages and keep the image size lower.

Do you know if it can be set in some config file as a default option?

Reply:
https://superuser.com/a/615583

I have not tried myself if this indeed works.